tests/test_facets.py

```diff
@@ -39,7 +39,7 @@ def test_facetbrowser(self):
         #wait for the facetbrowser to be loaded
         wait.until(expected_conditions.presence_of_element_located((By.ID, "bibdk-facetbrowser-form")))
 
-        harry_warren = browser.find_element(By.PARTIAL_LINK_TEXT, "harry edison")
+        harry_warren = browser.find_element(By.PARTIAL_LINK_TEXT, "harry james")
```

Comment:
Syret - denne har jeg netop lavet om til warren fra noget andet (i en feature branch), fordi den også fejlede for mig - parametre hedder harry_warren, mon ikke der er en grund til det?